src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -28,18 +28,21 @@ public void addGame(Game game){
     }
 
     public Game getGameByCode(String gameCode){
-        return games.stream().filter(g -> g.getGameCode().equals(gameCode)).findAny().get();
+        return games.stream()
+                .filter(g -> g.getGameCode().equals(gameCode))
+                .findAny()
+                .get();
     }
 
-    public void joinGame(String gameCode, Player playerToJoin){
-        getGameByCode(gameCode).getPlayers().add(playerToJoin);
+    public void joinGame(
+            String gameCode, Player playerToJoin)  {
+        getGameByCode(gameCode)
+                .getPlayers()
+                .add(playerToJoin);
     }
 
     public static synchronized GameTracker getInstance() {
-
-        if (gameTracker == null) {
-            gameTracker = new GameTracker();
-        }
+        gameTracker = (gameTracker != null) ? gameTracker : new GameTracker();
```

Comment:
refactored the "if" statement that assigned a value based on a boolean expression to use the Java Ternary Operator.
This is recommend by the "Ternary Operator" section of the java coding best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3